alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/roso-details/roso-details.component.html

```diff
@@ -78,16 +78,9 @@
     </div>
   </div>
 
-  <div class="full-width">
-    <h4>Project Duration</h4>
-  </div>
-  <div class="subheading2 grid-1">Duration</div>
-  <div class="grid-double">
-    {{ _noiSubmission.soilProjectDurationUnit }}
-  </div>
-  <div class="subheading2 grid-1">Length</div>
+  <div class="subheading2 grid-1">Removal of Soil Project Duration</div>
   <div class="grid-double">
-    {{ _noiSubmission.soilProjectDurationAmount }}
+    {{ _noiSubmission.soilProjectDuration }}
   </div>
```

Comment:
Move project duration field to before soil table to match rest of the soil types (and review page)

Reply:
moved